backend/server.py

```
@@ -4816,6 +4875,13 @@ async def get_stats(user: dict = Depends(get_current_user)):
     }
 
 
+@app.get("/api/stats")
+async def get_stats(user: dict = Depends(get_current_user)):
```

Ruff (0.15.17)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)